.changeset/dull-maps-draw.md

```diff
@@ -0,0 +1,8 @@
+---
+'@ts-rest/nest': minor
+'@ts-rest/core': patch
+---
+
+Rename some Nest functions and types, and deprecate old names
```

Comment:
Since we're renaming, should we indicate breaking changes in the release notes?

Reply:
We're still keeping the old names, but marking them as deprecated so it's technically not a breaking change yet

Reply:
AH missed that, thanks!